FabulousContacts/Helpers.fs

```diff
@@ -53,14 +53,12 @@ module Helpers =
 
     let takePictureAsync() = async {
         let options = new StoreCameraMediaOptions()
-        options.PhotoSize <- PhotoSize.MaxWidthHeight
         options.MaxWidthHeight <- Nullable<int>(150)
```

Comment:
In theory we could also remove this line (and the one below) of code, then?

Reply:
Yes, we can do that for now.
Until things improve in the Media.Plugin part

Reply:
I update the code, cheers!